Cognite.Common/FailureThresholdManager.cs

```diff
@@ -14,6 +14,12 @@ namespace Cognite.Extractor.Common
     public class FailureThresholdManager<T> where T : IComparable
     {
         private ConcurrentDictionary<T, byte> _failedJobs = new ConcurrentDictionary<T, byte>();
+
+        /// <summary>
+        /// Get failed jobs
+        /// </summary>
+        public HashSet<T> FailedJobs => new HashSet<T>(_failedJobs.Keys);
```

Comment:
This can be an enumerable

Reply:
Agree, ends up simpler https://github.com/cognitedata/dotnet-extractor-utils/pull/265/commits/15a777acad1136dca450d431368ab8658330725d